examples.md

```diff
@@ -170,6 +171,33 @@ The yarn cache directory will depend on your operating system and version of `ya
       ${{ runner.os }}-yarn-
 ```
 
+## OCaml - esy
+Esy allows you to export built dependencies and import pre-built dependencies.
+```yaml
+    - name: Restore Cache
+      id: restore-cache
+      uses: actions/cache@v1.1.2
```

Comment:
```suggestion
      uses: actions/cache@v1
```
The `@v1` tag is recommended as it tracks the latest v1 version specified

Reply:
Good to know, fixed it :)